spec/facades/parks_facade_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe ParksFacade do
+  it "creates park poros", :vcr do
+    park = ParksFacade.create_park("TN")
+    expect(park).to be_an(Array)
```

Comment:
maybe `parks` is better cause it's a collection, right?
You might also want to check that all the parks are of type Park

Given that your facade has two methods, you also will want to test the other method and make sure it returns  a total